packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts

```diff
@@ -1119,7 +1119,9 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
             rate: parseInt(quote.price)
           })
         )
-        const refresh = -moment().add(10, 'seconds').diff(quote.quoteExpiresAt)
+        const refresh = Math.abs(
+          differenceInMilliseconds(new Date(quote.quoteExpiresAt), addSeconds(new Date(), 10))
+        )
```

Comment:
* -1 and Math.abs are not equal, is this what you want?

Reply:
discussed offline. we are okay here